src/js/models/cpu_list_model.js

```diff
@@ -173,6 +173,13 @@ YUI.add('cpu-list-model', function(Y) {
                         var cpuData = item.cpuData; // the N CPUs, M values per CPU
                         var readTime = item.readTime;
 
+                        // just in case there is somehow a duplicate of KG-14770
+                        // that shows up for System data, put in a check that
+                        // cpuData is actually non-null.
+                        if (cpuData == null || cpuData == undefined) {
```

Comment:
I'm not expert, but wouldn't you want this to be

if (cpuData === undefined || cpuData == null)

instead?